styles.css

```diff
@@ -115,16 +106,32 @@ footer {
     margin-top: 4rem;
 }
 
+.footer-logo {
+    align-self: center
+}
+
 .links {
     list-style: none;
 }
 
-.links>li{
+.links>li {
     padding: 4px 0;
-    font-size: 1.1rem;
 }
 
 .link {
     color: var(--light-black);
     text-decoration: none;
+}
+
+
+@media screen and (max-width: 720px) {
+    .myhamburger {
+        display: block;
+    }
+}
+
+@media screen and (max-width: 480px) {
+    .sub-navigation {
+        display: none;
+    }
 }
```

Comment:
Here also add a new line.

Reply:
Prettier file formatter removes blank lines at the end